xla/service/cpu/onednn_ops_rewriter.cc

```diff
@@ -88,28 +146,33 @@ std::optional<HloInstruction*> MatchSoftmax(HloInstruction* instr) {
   // |   /
   // divide  // (instr parameter)
   //
-  // where both reductions occur only on the last axis.
+
+  // Though this is most common SoftMax pattern we see and The patterns added
```

Comment:
The capitalization is not correct here, please make sure to follow the classical grammatical and typographical rules for English language.

But thinking a bit deeper, I think what you are saying can be summarized with just the following sentence:

```
This matcher covers the most common SoftMax patterns we have encountered in real-life models.
```

(unless I missed something).